stackstorm.yml

```diff
@@ -3,12 +3,17 @@
   hosts: all
   environment: "{{ st2_proxy_env | default({}) }}"
   roles:
-    - StackStorm.mongodb
+    - role: StackStorm.mongodb
+      mongodb_version: 4.0
+      mongodb_apt_keys:
+        "3.2": "42F3E95A2C4F08279C4960ADD68FA50FEA312927"
+        "3.4": "0C49F3730359A14518585931BC711F9BA15703C6"
+        "4.0": "9da31620334bd75d9dcb49f368818c72e52529d4"
```

Comment:
Please move mongodb 4.0 apt key to role defaults:
https://github.com/StackStorm/ansible-st2/blob/45a463fdb4bce63be02b38624eb46d7ea7350952/roles/StackStorm.mongodb/vars/main.yml#L1-L6